themes/agnoster.zsh-theme

```diff
@@ -26,7 +26,13 @@
 # A few utility functions to make it easy and re-usable to draw segmented prompts
 
 CURRENT_BG='NONE'
-SEGMENT_SEPARATOR=''
+
+# Fix odd char on mac
+if [[ `uname` == 'Darwin' ]]; then
+    SEGMENT_SEPARATOR='\ue0b0'
```

Comment:
How old is your Powerline font? `\ue0b0` is the segment separator that Powerline currently uses. They changed the code points they were using a while back, and `\ue0b0` is the new one. (I can't tell which other character you're referring to, because my browser isn't using a powerline-patched font. And since there are two versions of the font, pasting the characters directly as single Unicode characters leads to ambiguity, because their appearance is going to depend on which versions of the font people in the conversation have installed.)